docs/Makefile

```diff
@@ -49,12 +49,21 @@ help:
 	@echo "  coverage   to run coverage check of the documentation (if enabled)"
 
 clean:
-	rm -rf $(BUILDDIR)/*
+	@rm -rf $(BUILDDIR)/*
 
-html:
+clean_rst: clean
```

Comment:
`clean_rst` will reset the RST file folders.

Reply:
question, non blocking

does it make more sense for `clean` to call `clean_rst` rather than the other way around?